smartcontractkit/external-adapters-js
# Update Synthetix Debt Pool adapter to reflect changes in SIP-165

## Closes 14251

## Description

This PR updates the Synthetix debt pool adapter to reflect the changes in SIP-165.  The pseudocode for these changes are outlined here https://sips.synthetix.io/sips/sip-165/.  

## Steps to Test

1) Run adapter and send a request to it
2) Verify it successfully returns a response and verify that the hex values match with what is expected from the SIP.

## Quality Assurance
